src/karma-webpack.js

```diff
@@ -69,7 +69,7 @@ function Plugin(
     // Must have the common _karma_webpack_ prefix on path here to avoid
     // https://github.com/webpack/webpack/issues/645
     webpackOptions.output.path = path.join(os.tmpdir(), '_karma_webpack_', indexPath, '/')
-    webpackOptions.output.publicPath = path.join(os.tmpdir(), '_karma_webpack_', publicPath, '/')
+    webpackOptions.output.publicPath = path.join('/_karma_webpack_', publicPath, '/')
```

Comment:
That was simply incorrect right (in general)? Resulting in async chunks loading to fail? I'm only an admin here with limited knowledge about the current needs of a `karma` setup, so I need to rely on your help here :)

Reply:
"That is simply incorrect right (in general)?" What do you mean by that? I believe this has been broken since PR #279 but has flown under the radar since it only breaks assets with strict CORS checking.